w3-aggregation.md

```diff
@@ -179,7 +179,7 @@ Invoking `aggregate/offer` capability submits an aggregate to a broker service f
 ]
 ```
 
-Each entry of the decoded offers block, has all the necessary information for a Storage Provider to fetch and store a CAR file. The `link` field has the CAR CID, while the `commitmentProof` field has the required `proof` bytes by Storage Providers (for example, `commP`). The `src` field of each piece MUST be set to a (alphabetically sorted) list of URLs from which it can be fetched. The `size` field MUST be set to the byte size of the CAR file.
+Each entry of the decoded offers block, has all the necessary information for a Storage Provider to fetch and store a CAR file. The `link` field has the CAR CID, while the `commitmentProof` field has the required `proof` bytes by Storage Providers (for example, `commP`). The `size` field MUST be set to the byte size of the CAR file. The `src` field of each piece MUST be set to a (alphabetically sorted) list of URLs from which it can be fetched. Note that `src` field is optional and can be provided in a different part of the flow such as when deal is signed or through a previously agreed API.
```

Comment:
Hey folks, this is concerning. The `byte`/`payload` size of the CAR file is not something ♠️ cares about nor can validate in any shape or form. Let's remove this and other `payload size` mentions from the spec 😅

Reply:
@ribasushi good callout. It is true that Spade does not care about this and won't validate. 

We can validate this in our side if we think that is important (perform HEAD requests to the src URLs). Main goal of having the size in an invocation is that it allows us to track metrics for data that we are requesting to Spade over time. All our metrics are being built around the information within invocations. 

With that in mind, is there any strong opinion against keeping size?

Reply:
You can keep it on your end, but there is very strong desire on the side of ♠️ to never receive/parse/store this information, and thus never be able to return it back to you on request.

So having something as a `MUST` when on my end I have `MUST DISCARD` is... odd ;)

Reply:
yeah, spade side won't need to do anything at it, even at the `spade-proxy` side. Just our UCAN Stream log processor will inspect these UCAN invocations and compute metrics over time from them.